code/datums/components/food/edible.dm

```diff
@@ -206,9 +208,16 @@ Behavior that's still missing from this component that original food items had t
 	var/atom/this_food = parent
 
 	this_food.reagents.multiply_reagents(CRAFTED_FOOD_BASE_REAGENT_MODIFIER)
+	this_food.reagents.maximum_volume *= CRAFTED_FOOD_BASE_REAGENT_MODIFIER
```

Comment:
wouldnt you want to adjust its volume before you affect the reagents in it

Reply:
Yes. It will work as it is because the multiplier is less than 1 but if someone changes it in the future it could break. I'll fix it.

Reply:
Actually, I can't because adding the necessary if statement results in a constant control flow evaluation, which the linter does not like.